wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1290,6 +1288,32 @@ def formattedfield(
     }
 
 
+@register.inclusion_tag("wagtailadmin/shared/formatted_field.html", takes_context=True)
+def formattedfieldfromcontext(context):
+    """
+    Variant of formattedfield that takes its arguments from the template context. Used by the
+    wagtailadmin/shared/field.html template.
+    """
+    kwargs = {}
+    for arg in (
```

Comment:
A matter of preference but we could define the args in a list and use dict comprehension to make this more declarative :D